crates/cli/src/subcommands/identity.rs

```diff
@@ -628,8 +628,8 @@ async fn exec_recover(mut config: Config, args: &ArgMatches) -> Result<(), anyho
     }
 
     let client = reqwest::Client::new();
-    let builder = client.get(Url::parse_with_params(
-        format!("{}/database/request_recovery_code", config.get_host_url(server)?,).as_str(),
+    let builder = client.post(Url::parse_with_params(
+        format!("{}/identity/request_recovery_code", config.get_host_url(server)?,).as_str(),
```

Comment:
shouldn't we also be updating `confirm_recovery_code`?

Reply:
Good catch